src/Controller/SpaceController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\SearchType;
+use App\Repository\SpaceRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class SpaceController extends AbstractController
+{
+
+    /**
+     * @Route("/space/{location}", name="app_space_list")
```

Comment:
location should be part of the query string, not an url parameter

Reply:
done